src/packet_parser/packet_parser.go

```diff
@@ -44,8 +46,8 @@ func (parser *PacketParser) AddPacket(globalInfo excelAdapterModels.GlobalInfo,
 	parser.descriptors[uint16(id)] = valueDescriptors
 }
 
-func NewPacketParser() PacketParser {
-	return PacketParser{
+func NewPacketParser() *PacketParser {
```

Comment:
Why pointer?

Reply:
AddGlobal and AddPacket work on a reference